test/data/notificationSourceTransferAtomic_TwoCases.json

```diff
@@ -0,0 +1,72 @@
+{
+  "id": "http://eur-ledger.example/EUR/subscriptions/52a42d6f-8d9c-4c05-b31c-cccc8bbdb31d",
+  "event": "transfer.update",
+  "resource": {
+    "id": "http://usd-ledger.example/USD/transfers/6851929f-5a91-4d02-b9f4-4ae6b7f1768c",
+    "ledger":"http://usd-ledger.example/USD",
+    "debits":[{
+      "amount":"1.07",
+      "account":"http://usd-ledger.example/accounts/alice"
+    }],
+    "credits":[{
+      "amount":"1.07",
+      "account":"http://usd-ledger.example/accounts/mark",
+      "memo":{
+        "destination_transfer": {
+          "id": "http://eur-ledger.example/EUR/transfers/c92f2a2c-b21d-4e6c-96e7-4f6d6df4bee9",
+          "ledger":"http://eur-ledger.example/EUR",
+          "debits":[{
+            "amount":"1.00",
+            "account":"http://eur-ledger.example/accounts/mark",
+            "memo": {
+              "source_transfer_ledger": "http://usd-ledger.example/USD",
+              "source_transfer_id": "http://usd-ledger.example/USD/transfers/6851929f-5a91-4d02-b9f4-4ae6b7f1768c"
+            }
+          }],
+          "credits":[{
+            "amount":"1.00",
+            "account":"http://eur-ledger.example/accounts/bob"
+          }],
+          "execution_condition": {
+            "message_hash": "9EXcGgJa0zTIUxEQXaMKpvoGsNmYSxv8J6C12LBNA5hRMfmbwDCzAU4O14UKnqcIaQqMGULHLI+T6Nk3+qYd3Q==",
```

Comment:
We could definitely use a more compressed format for including the next transfer details in the memo because a lot of this information can be inferred instead of duplicated. But that's for another PR